notebooks/messageAB_WBCD.r

```diff
@@ -40,82 +44,215 @@ axis_text_size <- 20
+    coord_flip() +
+    theme_bw() +
+    ylab("F-score") +
+    xlab(NULL) +
+    scale_fill_manual(
+      name = NULL,
+      values = cols,
+      # breaks = c(
+      #   paste0("Original (AUC = ", round(mean(scores_and_auc$auc_original) * 100, 2), ")"),
+      #   paste0("Avatar (AUC = ", round(mean(scores_and_auc$auc_synthetic) * 100, 2), ")"))
+    ) +
+    theme(
+      legend.position = c(0.72, 0.1),
+      legend.title = element_blank(),
+      legend.key.size = unit(0.8, "cm"),
+      legend.text = element_text(size = legend_text_size, color = "black", family = ""),
+      axis.text.x = element_text(size = axis_text_size, color = "black", family = ""),
+      axis.text.y = element_text(size = axis_text_size, color = "black", family = ""),
+      axis.title = element_text(size = axis_title_size, color = "black", family = ""),
+      axis.ticks.y = element_blank()
+    )
+  return(plot)
+}
 
-plotAd <- ggplot(data = df_scores_70, aes(x = reorder(feature, order), y = mean, fill = type)) +
-  geom_bar(stat = "identity", position = position_dodge(), color = "black") +
-  geom_errorbar(aes(ymin = mean - sd, ymax = mean + sd), width = 0, lwd = 1, position = position_dodge(.9)) +
-  coord_flip() +
+f1_scores_comparison <- read.csv("../datasets/results_df/comparative_f1_scores.csv")
+predictions_comparison <- read.csv("../datasets/results_df/comparative_predictions.csv")
+##### ORIGINAL - AVATAR comparison
+
+f1_scores_avatar <- subset(f1_scores_comparison, type == "Original" | type == "Avatar")
+predictions_avatar <- subset(predictions_comparison, type == "Original" | type == "Avatar")
+scores_and_auc_avatar <- reshape_predictions_and_f1score(predictions_avatar, f1_scores_avatar, synthetic_name = "Avatar")
+plotAd <- get_f1score_plot(scores_and_auc_avatar, synthetic_color = "avatar")
+
+
+##### ORIGINAL - Synthpop comparison
+
+f1_scores_synthpop <- subset(f1_scores_comparison, type == "Original" | type == "Synthpop")
+predictions_synthpop <- subset(predictions_comparison, type == "Original" | type == "Synthpop")
+scores_and_auc_synthpop <- reshape_predictions_and_f1score(predictions_synthpop, f1_scores_synthpop, synthetic_name = "Synthpop")
+f1score_plot_synthpop <- get_f1score_plot(scores_and_auc_synthpop, synthetic_color = "synthpop")
+
+##### ORIGINAL - ctgan comparison
+
+f1_scores_ctgan <- subset(f1_scores_comparison, type == "Original" | type == "CT-GAN")
+predictions_ctgan <- subset(predictions_comparison, type == "Original" | type == "CT-GAN")
+scores_and_auc_ctgan <- reshape_predictions_and_f1score(predictions_ctgan, f1_scores_ctgan, synthetic_name = "CT-GAN")
+f1score_plot_ctgan <- get_f1score_plot(scores_and_auc_ctgan, synthetic_color = "ctgan")
+
+## Comparative utility plot 
+vec <- c("Bare Nuclei", "Uniformity of Cell Shape", "Uniformity of Cell Size",
+         "Bland Chromatin", "Clump Thickness", "Normal Nucleoli", "Marginal Adhesion",
+         "Single Epithelial Cell Size", "Mitoses") 
```

Comment:
nitpick: we should avoid hard coding, 
```suggestion
vec <- rownames(wbcd) # or something similar
```